HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5630,6 +5734,40 @@ def self.get_battery_values(has_garage)
              usable_fraction: 0.9 } # Fraction of usable capacity to nominal capacity
   end
 
+  # Get default lifetime model, miles/year, hours/week, nominal capacity/voltage, round trip efficiency, fraction charged at home,
+  # and usable fraction for an electric vehicle and its battery.
+  #
+  # @return [Hash] map of EV properties to default values
+  def self.get_eletric_vehicle_values()
+    return { battery_type: HPXML::BatteryTypeLithiumIon,
+             lifetime_model: HPXML::BatteryLifetimeModelNone,
+             miles_per_year: 10900,
+             hours_per_week: 8.88,
+             nominal_capacity_kwh: 63,
+             nominal_voltage: 50.0,
+             fuel_economy: 0.22,
+             fuel_economy_units: HPXML::UnitsKwhPerMile,
+             fraction_charged_home: 0.8,
+             usable_fraction: 0.8 } # Fraction of usable capacity to nominal capacity
+  end
+
+  # Get default location, charging power, and charging level for an electric vehicle charger.
+  # The default location is the garage if one is present.
+  #
+  # @param has_garage [Boolean] whether the HPXML Building object has a garage
+  # @return [Hash] map of electric vehicle charger properties to default values
+  def self.get_ev_charger_values(has_garage = false)
+    if has_garage
+      location = HPXML::LocationGarage
+    else
+      location = HPXML::LocationOutside
+    end
+
+    return { location: location,
+             charging_power: 5690, # Median L2 charging rate in EVWatts
+             charging_level: 2 }
```

Comment:
We should really use the HPXML `ChargingLevel` element. We could get an HPXML file where someone has told us there is a Level 1 charger but then we ignore it and assume charging power associated with Level 2, that's not great. (This would also require updating schematron, documentation, etc.)

Reply:
We don't ever write or use the charging level, so this value is obsolete, I deleted it.